docs/manual/08_Commands.md

```diff
@@ -0,0 +1,372 @@
+# Commands
+Clusterous is a command line tool to manage virtual machines on Amazon Web Services (AWS).
```

Comment:
That's probably not an accurate description. Perhaps just use "Tool to create and manage compute clusters" from the help output.